excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/level/MapParserTest.java

```diff
@@ -61,12 +60,6 @@ public static Collection data() {
                 GameElementType.OBSTACLE,
                 GameElementType.LINE_CHANGER_UP,
                 GameElementType.LINE_CHANGER_DOWN,
-                SpringboardElementType.SPRINGBOARD_DARK,
```

Comment:
Why do you have removed those test cases? They are still valid, no?

Reply:
When I made springboard element as complex element, I had removed this case, now I have returned it and added other springboard element's types to the test.